docs/tutorials/09_Protein_Folding.ipynb

```diff
@@ -0,0 +1,720 @@
+    "\n",
+    "Unfolded polypeptides have a very large number of degrees of freedom\n",
+    "and thus an enormous number of potential conformations. For example, a\n",
+    "chain with 100 aminoacids has on the order of 10<sup>47</sup> conformations. In\n",
+    "reality, however, many proteins fold to their native structure within\n",
+    "seconds. This is known as Levinthal’s paradox [1].\n",
+    "\n",
+    "The exponential growth of potential conformations with chain length\n",
+    "makes the problem intractable for classical computers. In the quantum\n",
+    "framework, our resource-efficient algorithm scales linearly with\n",
+    "the number of amino acids N.\n",
+    "\n",
+    "\n",
+    "<font size='2'>[1] https://en.wikipedia.org/wiki/Levinthal%27s_paradox </font>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3110941f",
+   "metadata": {},
+   "source": [
+    "The goal of this work is to determine the minimum energy conformation of a protein. Starting from a random configuration, the protein's structure is optimized to lower the energy. This can be achieved by encoding the protein folding problem into a qubit operator and ensuring that all physical constraints are satisfied. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "29deb98b",
+   "metadata": {},
+   "source": [
+    "For the problem encoding we use: \n",
+    "\n",
+    "- Configuration qubits: qubits that are used to describe the configurations and the relative position of the different beads\n",
+    "- Interaction qubits: qubits that encode interactions between the different aminoacids\n",
+    "\n",
+    "For our case we use a tetrahydral lattice (diamond shape lattice) where we encode the movement through the configuration qubits (see image below). \n",
+    "\n",
+    "<img src=\"aux_files/lattice_protein.png\" width=\"300\">"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f571d4a8",
+   "metadata": {},
+   "source": [
+    "The Hamiltonian of the system for a set of qubits $\\mathbf{q}=\\{\\mathbf{q}_{cf}, \\mathbf{q}_{in}\\}$ is \n",
+    "\n",
+    "$$H(\\mathbf{q}) = H_{gc}(\\mathbf{q}_{cf}) + H_{ch}(\\mathbf{q}_{cf}) + H_{in}(\\mathbf{q}) $$\n",
+    "\n",
+    "where \n",
+    "- $H_{gc}$ is the geometrical constraint term (governing the growth of the primary sequence of aminoacids without bifurcations\n",
+    "- $H_{ch}$ is the chirality constraint (enforcing the right stereochemistry for the system)\n",
+    "- $H_{in}$ is the interaction energy terms of the system. In our case we consider only nearest neighbor interactions. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "62d99a5f",
+   "metadata": {},
+   "source": [
+    "Further details about the used model and the encoding of the problem can be found in [2].\n",
+    "\n",
+    "<font size='2'>[2] A.Robert, P.Barkoutsos, S.Woerner and I.Tavernelli, Resource-efficient quantum algorithm for protein folding, NPJ Quantum Information, 2021 </font>"
```

Comment:
Is there a reason why these references are not collected together at the end of the notebook in a references section? Having them in among the text means you have to go look for them a bit, although like the above its so close and I think is the only reference the pointer could be much more direct.
I had to search hard in the html for [4] - turns out it wrapped in a paragraph and tagged directly on the end of [3] so was not so easy to find. Having a references section with these as the bullets, if they are not wanted inline, would make it so much easier I think.

Reply:
I added a section with all the references. I think it is now easier to read.